viz.Rmd

```diff
@@ -716,3 +1089,12 @@ knitr::include_graphics("img/faithful_zoom.svg")
 - The [`ggplot2` page on the tidyverse website](https://ggplot2.tidyverse.org) is where you should look if you want to learn more about the functions in this chapter, the full set of arguments you can use, and other related functions. The site also provides a very nice cheat sheet that summarizes many of the data wrangling functions from this chapter.
```

Comment:
should provide the links here -- I guess this will be a part of the formatting pass

Reply:
yep